clang/test/Analysis/Checkers/WebKit/call-args-safe-functions.cpp

```diff
@@ -44,6 +43,12 @@ inline Target* uncheckedDowncast(Source* source)
     return static_cast<Target*>(source);
 }
 
+template<typename Target, typename Source>
+Target* [[clang::annotate_type("webkit.pointerconversion")]] newCastFunction(Source*);
```

Comment:
If this function is a member function, will you accidentally take the implicit `this` object as the original argument?

Reply:
I don't think so. I added a test case for that.